mmv1/products/dialogflow/api.yaml

```diff
@@ -334,58 +332,543 @@ objects:
+                    - !ruby/object:Api::Type::NestedObject
+                      name: "queryConfig"
+                      description: |
+                        Configs of query.
+                      properties:
+                        - !ruby/object:Api::Type::Integer
+                          name: "maxResults"
+                          description: |
+                            Maximum number of results to return. Currently, if unset, defaults to 10. And the max number is 20.
+                        - !ruby/object:Api::Type::Double
+                          name: "confidenceThreshold"
+                          description: |
+                            Confidence threshold of query result.
+                            Agent Assist gives each suggestion a score in the range [0.0, 1.0], based on the relevance between the suggestion and the current conversation context. A score of 0.0 has no relevance, while a score of 1.0 has high relevance. Only suggestions with a score greater than or equal to the value of this field are included in the results.
+                            For a baseline model (the default), the recommended value is in the range [0.05, 0.1].
+                            For a custom model, there is no recommended value. Tune this value by starting from a very low value and slowly increasing until you have desired results.
+                            If this field is not set, it defaults to 0.0, which means that all suggestions are returned.
+                            Supported features: ARTICLE_SUGGESTION, FAQ, SMART_REPLY, SMART_COMPOSE.
+                        - !ruby/object:Api::Type::NestedObject
+                          name: "contextFilterSettings"
+                          description: |
+                            Determines how recent conversation context is filtered when generating suggestions. If unspecified, no messages will be dropped.
+                          properties:
+                            - !ruby/object:Api::Type::Boolean
+                              name: "dropHandoffMessages"
+                              description: |
+                                If set to true, the last message from virtual agent (hand off message) and the message before it (trigger message of hand off) are dropped.
+                            - !ruby/object:Api::Type::Boolean
+                              name: "dropVirtualAgentMessages"
+                              description: |
+                                If set to true, all messages from virtual agent are dropped.
+                            - !ruby/object:Api::Type::Boolean
+                              name: "dropIvrMessages"
+                              description: |
+                                If set to true, all messages from ivr stage are dropped.
+                        - !ruby/object:Api::Type::NestedObject
+                          name: "knowledgeBaseQuerySource"
```

Comment:
Based on the [API documentation](https://cloud.google.com/dialogflow/es/docs/reference/rest/v2/projects.conversationProfiles#ConversationProfile.SuggestionQueryConfig), it looks like Union field `query_source` can be only one of the following:
- knowledgeBaseQuerySource
- documentQuerySource
- dialogflowQuerySource
We need to add `conflicts` for these fields.